Remove parenthesis from id slugs

timgws opened the pull request:
Two small changes.

* Remove any parenthesis from the generated slugs
* Add markdown-include, so that other Markdown pages can be included from the index (and other pages, too!) by using this syntax:
```
!!!include(notice.api-key.md)!!!
```

MikeRalphson commented:
Sorry to be a pain, and thanks for the PR. But could you omit the `!!!include` change for now? I'd like to keep reslate functionally compatible with Ruby Slate and not introduce different include syntax until such time as reslate becomes the official Slate v3.

The slug parenthesis change LGTM.

timgws commented:
Apologies for taking my sweet time to get this to you. I have updated the commit to remove the `include` syntax, and updated the commit message.

I'll keep a keen eye for when reslate looks to become the official slate replacement, to try and get it added :)

MikeRalphson commented on .eleventy.js:
```diff
@@ -16,7 +24,9 @@ module.exports = function(eleventyConfig) {
       typographer: true
-    }).use(markdownItAnchor, {})
+    }).use(markdownItAnchor, {
+        slugify: slugify,
```
```suggestion
        slugify: slugify
```

MikeRalphson reviewed (approved):
Compared with Ruby Slate and that does remove parentheses.
